contxt/services/nionic.py

```diff
@@ -15,27 +16,20 @@ class NionicService(BaseGraphService):
     _envs = (
         ApiEnvironment(
             name="production",
-            base_url="https://<tenant>.api.ndustrial.io",
+            base_url="https://{tenant}.api.ndustrial.io",
```

Comment:
This is the base url for all of them.  If it is more convenient to use it.

Reply:
Ah of course! 🤦
Thanks!